pkg/envoy/server.go

```diff
@@ -182,7 +182,8 @@ func StartXDSServer(stateDir string) *XDSServer {
 											"prefix": {Kind: &structpb.Value_StringValue{StringValue: "/"}},
 										}}}},
 										"route": {Kind: &structpb.Value_StructValue{StructValue: &structpb.Struct{Fields: map[string]*structpb.Value{
-											"cluster": {Kind: &structpb.Value_StringValue{StringValue: "cluster1"}},
+											"cluster":          {Kind: &structpb.Value_StringValue{StringValue: "cluster1"}},
+											"max_grpc_timeout": {Kind: &structpb.Value_StructValue{StructValue: &structpb.Struct{Fields: map[string]*structpb.Value{}}}},
```

Comment:
I'm curious what would happen if you set this option but the sidecar proxies are still running older versions of Envoy, which don't support this option?
I.e. are we backward compatible?
Could you please give it a shot, e.g. temporarily revert the Envoy SHA update and see whether Envoy barfs on that field?

Reply:
Oh, I'm stupid. This is irrelevant for sidecar proxies, since their configuration will come from pilot-agent and/or pilot. Nevermind.